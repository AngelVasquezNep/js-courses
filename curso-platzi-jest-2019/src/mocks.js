const CHARACTERS = {
  rick: {
    id: 1,
    name: "Rick Sanchez",
    status: "Alive",
    species: "Human",
    type: "",
    gender: "Male",
  },
};

export default {
  CHARACTERS,
};
