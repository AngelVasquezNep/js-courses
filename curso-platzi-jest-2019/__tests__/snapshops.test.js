import mocks from '../src/mocks';

describe("Snapshots", () => {
  test("Rick has right surname", () => {
    expect(mocks.CHARACTERS.rick).toMatchSnapshot()
  })
})