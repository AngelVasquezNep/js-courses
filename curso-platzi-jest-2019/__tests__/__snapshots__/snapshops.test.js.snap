// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Rick has right surname 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
  "type": "",
}
`;
